BloodPlusFrontEnd/donator/src/css/TextField.css

```diff
@@ -0,0 +1,344 @@
+}
+
+.input__field--hoshi + .input__label--hoshi .input__label-content--hoshi,
+.input--filled .input__label-content--hoshi {
+    -webkit-animation: anim-1 0.3s forwards;
+    animation: anim-1 0.3s forwards;
+}
+
+@-webkit-keyframes anim-1 {
+    50% {
+        opacity: 0;
+        -webkit-transform: translate3d(1em, 0, 0);
+        transform: translate3d(1em, 0, 0);
+    }
+    51% {
+        opacity: 0;
+        -webkit-transform: translate3d(-1em, -40%, 0);
+        transform: translate3d(-1em, -40%, 0);
+    }
+    100% {
+        opacity: 1;
+        -webkit-transform: translate3d(0, -40%, 0);
+        transform: translate3d(0, -40%, 0);
+    }
+}
+
+@keyframes anim-1 {
+    50% {
+        opacity: 0;
+        -webkit-transform: translate3d(1em, 0, 0);
+        transform: translate3d(1em, 0, 0);
+    }
+    51% {
+        opacity: 0;
+        -webkit-transform: translate3d(-1em, -40%, 0);
+        transform: translate3d(-1em, -40%, 0);
+    }
+    100% {
+        opacity: 1;
+        -webkit-transform: translate3d(0, -40%, 0);
+        transform: translate3d(0, -40%, 0);
+    }
+}
+.row {
+  margin-left: -15px;
+  margin-right: -15px;
+}
+
+.container-fluid {
+  margin-right: auto;
+  margin-left: auto;
+  padding-left: 15px;
+  padding-right: 15px;
+}
+body {
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-size: 14px;
+  line-height: 1.42857143;
+  color: #333333;
+  background-color: #fff;
+}
+
+img {
+  vertical-align: middle;
+}
+
+.img-responsive {
+  display: block;
+  max-width: 100%;
+  height: auto;
+
+}
+.img-rounded {
+  border-radius: 6px;
+}
+
+.img-thumbnail {
+  padding: 4px;
+  line-height: 1.42857143;
+  background-color: #fff;
+  border: 1px solid #ddd;
+  border-radius: 4px;
+  -webkit-transition: all 0.2s ease-in-out;
+  -o-transition: all 0.2s ease-in-out;
+  transition: all 0.2s ease-in-out;
+  display: inline-block;
+  max-width: 100%;
+  height: auto;
+}
+
+.img-circle {
+  border-radius: 50%;
+}
+
+.right-side {
+  position: relative;
+  padding: 50px 120px 100px 120px;
+  height: 100vh;
+  background: #fff;
+}
+
+.right-side .input {
+  margin-top: 0;
+  margin-left: 0;
+}
+.right-side .input input {
+  font-size: 14px;
+}
+.right-side .cta {
+  margin-top: 40px;
+}
+.right-side .cta .btn {
+  padding: 13px 25px;
+  border: none;
+  color: #fff;
+  display: inline-block;
+  background: #ff5500;
+  font-family: 'lato', sans-serif;
+  text-transform: uppercase;
+  font-size: 13px;
+  letter-spacing: 0.12em;
+  border-radius: 24px;
+}
+.right-side .cta .btn:hover {
+  -webkit-transition: 0.2s ease-in;
+  -o-transition: 0.2s ease-in;
+  transition: 0.2s ease-in;
+  background: #ff3f00;
+}
+.right-side .cta span {
+  margin-top: 10px;
+  margin-left: 30px;
+  display: inline-block;
+}
+.right-side .cta span a {
+  font-family: 'lato', sans-serif;
+  text-transform: uppercase;
+  font-size: 13px;
+  color: #999;
+  letter-spacing: 0.1em;
+}
+
+input[type=radio] {
```

Comment:
Sa incercam sa evitam css cu selector general pentru ca o sa se propage in alte pagini.
 Poti sterge css care nu e folosit, de exemplu pentru radio button.